Lib/pdb.py

```diff
@@ -418,6 +419,16 @@ def setup(self, f, tb):
             ]
             self.rcLines = []
 
+    @property
+    @deprecated('Use curframe.f_locals instead')
+    def curframe_locals(self):
+        return self.curframe.f_locals
+
+    @curframe_locals.setter
+    @deprecated('Use curframe.f_locals instead')
```

Comment:
We should warn people that the setter no longer has any effect.

```suggestion
    @deprecated("Setting 'curframe_locals' no longer has any effect. Update the contents of 'curframe.f_locals' instead.")
```